packages/client/src/services/LoadCanaryConfigService.ts

```diff
@@ -39,6 +40,10 @@ export default class LoadCanaryConfigService {
       axios.get(`${process.env.PUBLIC_URL}/templates/${fileName}`).then(response => {
         configEditorStore.setCanaryConfigObject(response.data);
       });
+      metricsService.sendMetric('canaryConfigTool', `${templateName}`)
+    }
+    else {
+      metricsService.sendMetric('canaryConfigTool', '')
```

Comment:
I think with typescript / javascript this can be one line and you just make the last item `?:` null-able.